src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
Oh, and by the way it is no more assured that config has the correct content that the file.  Could just as easily be over writing good file with bad.